src/components/ArtistList/ArtistList.js

```diff
@@ -1,70 +1,56 @@
-import React, { Fragment, useEffect, useState } from "react";
+import React, { Fragment, useEffect} from "react";
+import { useSelector,useDispatch } from "react-redux";
+
 import Select from "react-dropdown-select";
 import ReactPaginate from "react-paginate";
 
+import { comboActions } from "../../store/combo-slice";
+import {fetchData} from '../../store/data-slice'
+
 import ArtistCard from "../ArtistCard/ArtistCard";
 import "./ArtistList.css";
 
 const ArtistList = () => {
-  //defining states
-  const [artists, setArtists] = useState([]);
-  const [pageCount, setPageCount] = useState(0);
-  const [itemsPerPage,setItemsPerPage]=useState(10)
-
-  const options=[{value:5,label:'five items per page'},{value:10,label:'ten items per page'},{value:15,label:'fifteen items per page'},{value:20,label:'twenty items per page'}]
-
-  //change items per page on select
-const itemsPerPageHandler=(selectedOption)=>{
-  setItemsPerPage(selectedOption[0].value)
-}
-
-
-  //for the first time that the page loads so we have something to show
-  //calculates the total number of pages we need for pagination
+  const dispatch=useDispatch()
+
```

Comment:
Do we need these blank lines here? (applies to all other files)

Reply:
code formated with prettier. blank lines removed.